ai_presenter/text_ai/chatgpt/chatgpt.py

```diff
@@ -1,11 +1,10 @@
 import openai
-import os
 import logging
 
 
 class ChatGPT:
     def __init__(self) -> None:
-        openai.api_key = os.getenv("CHATGPT_APIKEY")
```

Comment:
This file probably needs to be set back to the original. To do that you can do the following:

From the top of the directory tree of this code:

```
git checkout main -- ai_presenter/text_ai/chatgpt/chatgpt.py
```